src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -41,9 +42,15 @@ const ApplicationLauncher = _.flow(
   }) // TODO: Check if name: workspaceName could be moved into the other workspace deconstruction
 )(({ name: workspaceName, sparkInterface, refreshRuntimes, runtimes, persistentDisks, application, workspace, workspace: { workspace: { googleProject, bucketName } } }, _ref) => {
   const cookieReady = useStore(cookieReadyStore)
+  const signal = useCancellation()
+  const interval = useRef()
+  const { user: { email } } = useStore(authStore)
   const [showCreate, setShowCreate] = useState(false)
   const [busy, setBusy] = useState(false)
   const [location, setLocation] = useState(defaultLocation)
+  const [outdatedAnalyses, setOutdatedAnalyses] = useState()
+  const [fileOutdatedOpen, setFileOutdatedOpen] = useState(false)
+  const [hashedOwnerEmail, setHashedOwnerEmail] = useState()
```

Comment:
We typically list the `state`s on top for easier parsing since they tend to be the first part to look at when reading a component code:

```suggestion
  // State
  const [showCreate, setShowCreate] = useState(false)
  const [busy, setBusy] = useState(false)
  const [location, setLocation] = useState(defaultLocation)
  const [outdatedAnalyses, setOutdatedAnalyses] = useState()
  const [fileOutdatedOpen, setFileOutdatedOpen] = useState(false)
  const [hashedOwnerEmail, setHashedOwnerEmail] = useState()
  
  const cookieReady = useStore(cookieReadyStore)
  const signal = useCancellation()
  const interval = useRef()
  const { user: { email } } = useStore(authStore)
```